packages/state-importer/src/main.js

```diff
@@ -136,21 +138,21 @@ export default class StateImport {
   _createOrUpdate(
     newStates: Array<StateData>,
     existingStates: Array<StateData>
-  ) {
+  ): Promise<any> {
```

Comment:
Looks like a lot of Promise<any> here which should not be. We should expand a bit on it and not use generic `any` all over the place.